ibmmq/README.md

```diff
@@ -0,0 +1,312 @@
+# IBM MQ Bindings
+
+This document defines how to describe IBM MQ specific information with AsyncAPI.
+
+<a name="version"></a>
+
+## Version
+
+Current version is `0.1.0`.
+
+## Terminology
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this bindings specification are to be interpreted as described in IETF [RFC2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+## AsyncAPI considerations
+
+This specification binding requests that the AsyncAPI specification reserve the IBM MQ protocol to allow connections to queue manager endpoints to be defined within the server object.
+
+AsyncAPI Object Fixed Field Name | Reserved Value for IBM MQ Protocol | Description
+---|:---|:---
+<a name="serverObjectProtocolFieldValueIbmmq"></a>`server.protocol` | ibmmq | IBM MQ protocol.
```

Comment:
@fmvilas shouldn't we consider this new protocol and binding as something reserved, like names we already have in the spec https://github.com/asyncapi/asyncapi/blob/master/versions/2.0.0/asyncapi.md#server-object and therefor this part should also be extended in the spec? I don't think it is must-have, but nice to have anyway?

Reply:
Absolutely. In AsyncAPI 1, protocols were reserved because the list was fixed. This changed in v2 and we made a free-form list, i.e., anyone can put there whatever they want. As long as tooling understands it, that's fine. It would be great to have a middle way between v1 and v2 and add some names to a reserved list of names 👍 Should we create a new issue maybe?

Reply:
@rcoppen would you mind adding a pointer to this document in the main spec? Otherwise, it will not be considered as part of the spec yet. You have to add links here:

* https://github.com/asyncapi/asyncapi/blob/master/versions/2.0.0/asyncapi.md#serverBindingsObject
* https://github.com/asyncapi/asyncapi/blob/master/versions/2.0.0/asyncapi.md#channel-bindings-object
* https://github.com/asyncapi/asyncapi/blob/master/versions/2.0.0/asyncapi.md#operation-bindings-object
* https://github.com/asyncapi/asyncapi/blob/master/versions/2.0.0/asyncapi.md#message-bindings-object

Just follow the same convention we used for the rest of the protocols.

Reply:
@fmvilas sure, no problem. Is there a convention to the ordering of the protocol list?

Reply:
Actually, it used to be sorted alphabetically but `mercure` got there at the end by mistake. If you can fix it at the same time that would be awesome.